stable_baselines/common/vec_env/base_vec_env.py

```diff
@@ -137,8 +137,11 @@ def step_wait(self):
     def close(self):
         return self.venv.close()
 
-    def render(self, mode='human'):
-        self.venv.render()
+    def render(self, **kwargs):
+        return self.venv.render(**kwargs)
+
+    def get_images(self):
```

Comment:
get_images is not defined in DummyVecEnv

Reply:
Done